src/registrar/assets/sass/_theme/_usa-modal.scss

```diff
@@ -0,0 +1,13 @@
+@use "cisa_colors" as *;
+@use "uswds-core" as *;
+
+.usa-modal.wider-modal {
+    width: 40% !important;
+    max-width: 40% !important;
+}
+
+.usa-modal {
+    .usa-modal__heading.blue-header{
+        color: $dhs-blue;
+    }
+}
```

Comment:
This is the same color as the figma but its not consistent with the other modals in our app (consistency difference between that and the figma). Thoughts?

Reply:
@zandercymatics sometimes I forget to update the color when pulling in the USWDS base modal component because the color is so similar to our .gov palette. Go ahead and match the color we have for our styles on this and any modals going forward.

Reply:
I double checked and I sometimes mistakenly use medium-50 when it should be dark-60. My bad!

Reply:
Not a problem!